examples/5_Pulse_programming_and_dynamical_decoupling.ipynb

```diff
@@ -86,7 +86,7 @@
     "\n",
     "@aq.main\n",
     "def idle():\n",
```

Comment:
I would do
```
drive_frame = device.frames[f"Transmon_{qubit}_charge_tx"]
readout_frame = device.frames[f"Transmon_{qubit}_readout_rx"]
pulse.delay(drive_frame, idle_duration)
pulse.capture_v0(readout_frame)
```
But I am not sure if it's equivalent to the one you have. It may need an experiment to find out.

Reply:
I tried this and it works (including running on the device). I don't think it matters if it's equivalent, since the previous code did not work at all.